rfcs/1-view-change-optimizations.md

```diff
@@ -0,0 +1,137 @@
+ * Start Date: 2017-05-10
+ * Haret issues: https://github.com/vmware/haret/issues/93
+
```

Comment:
since this RFC deals heavily with issues referencing the paper, it might be nice to provide a link.